src/presenters/admin/indexation-warning-presenter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Presenter class for the indexation warning.
+ *
+ * @package Yoast\YoastSEO\Presenters\Admin
+ */
+
+namespace Yoast\WP\SEO\Presenters\Admin;
+
+use Yoast\WP\SEO\Presenters\Abstract_Presenter;
+
+/**
+ * Indexation_Warning_Presenter class.
+ */
+class Indexation_Warning_Presenter extends Abstract_Presenter {
+
+	/**
+	 * Presents the warning that your site's content is not fully indexed.
+	 *
+	 * @return string The warning HTML.
+	 */
+	public function present() {
+		return \sprintf(
+			'<div id="yoast-indexation-warning" class="notice notice-warning"><p><strong>%1$s</strong> %2$s</p></div>',
+			\sprintf(
+				/* translators: 1: Link start tag to open the indexation modal, 2: Link closing tag. */
+				\esc_html__( 'Yoast SEO needs to index your site to have the best performance. Please %1$sindex your site now%2$s.', 'wordpress-seo' ),
```

Comment:
Has this copy been checked yet with product?

It is not a blocker, I just want to make sure that this is not forgotten 🙂.

Reply:
A few updates have been made after posting this in product. It's now completely up-to-date.